tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -342,6 +321,105 @@ def test_multiple_fsx(
     run_benchmarks(remote_command_executor, scheduler_commands)
 
 
+@pytest.mark.usefixtures("instance", "scheduler")
+def test_multi_az_fsx(
+    os,
+    region,
+    fsx_factory,
+    svm_factory,
+    open_zfs_volume_factory,
+    vpc_stack,
+    pcluster_config_reader,
+    clusters_factory,
+    s3_bucket_factory,
+    scheduler_commands_factory,
+    test_datadir,
+):
+    bucket_name = s3_bucket_factory()
+    bucket = boto3.resource("s3", region_name=region).Bucket(bucket_name)
+    bucket.upload_file(str(test_datadir / "s3_test_file"), "s3_test_file")
+    import_path = "s3://{0}".format(bucket_name)
+    export_path = "s3://{0}/export_dir".format(bucket_name)
+
+    partition = utils.get_arn_partition(region)
+    num_existing_fsx_ontap_volumes = 2 if partition != "aws-cn" else 0  # China does not have Ontap
```

Comment:
do we really need to test multiple FS of the same type here? I think this is already covered by the other test